routers/repo/reviews.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package repo
+
+import (
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/google/git-appraise/repository"
```

Comment:
Local imports (e.g.  `code.gitea.io/gitea/foo/bar`) should come before non-stdlib external packages (e.g. `github.com/foo/bar`).

```
import (
	"strconv"
	"strings"
	"time"

	"code.gitea.io/gitea/models"
	"code.gitea.io/gitea/modules/base"
	"code.gitea.io/gitea/modules/context"

	"github.com/google/git-appraise/repository"
	"github.com/google/git-appraise/review"
)
```